server/main.lua

```diff
@@ -42,6 +42,18 @@ end
 
 exports('ReleasePlayer', releasePlayer)
 
+RegisterNetEvent("qbx_prison:server:releasePlayer", function ()
+    local src = source
+    local Player = exports.qbx_core:GetPlayer(src)
+    if not Player then return end
+    if Player.Functions.metadata.injail and Player.Functions.metadata.injail == 0 then
```

Comment:
Make sure to test this. I'm concerned about a race condition where this doesn't get set by the time this check runs .